src/components/CardMap/CardList.js

```diff
@@ -1,12 +1,14 @@
 import Cards from '../Cards/Cards';
+import useStore from '../../hooks/useStore';
 
-export default function CardList({ postdata }) {
+export default function CardList() {
+	const cardlistzustand = useStore(state => state.cardlistzustand);
 	return (
 		<ul>
-			{postdata.map(data => {
+			{cardlistzustand.map(angebot => {
```

Comment:
I'd recommend to name all variables and object properties in english!

```suggestion
			{cardlistzustand.map(offer => {
```